tests/foreman/api/test_contentview.py

```diff
@@ -693,6 +700,92 @@ def test_positive_publish_with_puppet_multiple(self):
         for cvv in content_view.read().version:
             self.assertEqual(len(cvv.read().puppet_module), 1)
 
+    @tier2
+    def test_positive_publish_with_swid_tags(self):
+        """Verify SWID tags content file should exist in publish content view
+        version location
+
+        :id: 25f10e99-ceef-4543-9b9c-fc21690c088b
+
+        :steps:
+            1. create product and repository with custom contents having swid tags
+            2. sync the repository
+            3. create the content view
+            4. publish the content-view
+            5. ssh into Satellite
+            6. verify SWID tags content file exist in publish content view version location
+
+        :expectedresults: SWID tags content vile should exist in publish content view
+            version location
+
+        :CaseAutomation: Automated
+
+        :CaseImportance: High
+
+        :CaseLevel: Integration
+        """
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.repository = [self.swid_repo]
+        content_view = content_view.update(['repository'])
+        content_view.publish()
+        content_view = content_view.read()
+        content_view_version_info = content_view.version[0].read()
+        self.assertEqual(len(content_view.repository), 1)
+        self.assertEqual(len(content_view.version), 1)
```

Comment:
just a query .. can we verify the total count of swid_tags (I never see how it looks but just a thought)

Reply:
If you saying w.r.t to satellite, we don't store any information about swid tags in satellite database anywhere.

Reply:
okay ... thanks